theia-cpp-electron/package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "private": true,
+  "name": "@theia/cpp-electron-example",
+  "version": "0.0.1",
+  "description": "Theia C/C++ Electron example",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/theia-ide/theia-apps.git"
+  },
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/theia-ide/theia-apps/issues"
+  },
+  "homepage": "https://github.com/theia-ide/theia-apps/theia-cpp-electron",
+  "theia": {
+    "target": "electron",
+    "frontend": {
+      "config": {
+        "applicationName": "Theia C/C++ Electron Example",
+        "preferences": {
+          "files.enableTrash": false,
+          "cpp.clangdExecutable": "clangd",
+          "cpp.clangdArgs": "",
+          "cpp.clangTidy": false
+        }
+      }
+    }
+  },
+  "scripts": {
+    "build": "yarn theia rebuild:electron; yarn theia build",
+    "start": "yarn theia start"
+  },
+  "dependencies": {
+    "@theia/callhierarchy": "latest",
+    "@theia/console": "latest",
+    "@theia/core": "latest",
+    "@theia/cpp": "latest",
```

Comment:
Will we later include the debug extension?

Reply:
I assume so based on the [readme entry](https://github.com/theia-ide/theia-apps/pull/219/files#diff-f284d19834094e5dbee8197c08e58a61R51)